.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -16,13 +25,13 @@ body:
     attributes:
       label: What did you expect to happen?
     validations:
-      required: true
+      required: false
 
   - type: textarea
     id: repro
     attributes:
       label: How can we reproduce it (as minimally and precisely as possible)?
```

Comment:
```suggestion
      label: How can we reproduce it (as minimally and precisely as possible)? Screenshots and videos are useful
```

Reply:
I already specify in the description that if they have screens or videos then they can share it.

Reply:
I added this hear, because you added it for the section "What happend ?". If it is clear that you can add screens or videos also in this section, you can resolve my comment ! 😄